autobuild/workflow.md

```diff
@@ -0,0 +1,195 @@
+```
+
+**Output:** updated manifest-latest.json
+
+### utils/create-j2-confs.py
+**Purpose:** create j2 skeletons from manifest-latest.json and wallet/xbridge config fragments. This is mainly intended to be a one-time operation to convert the old style two-files per coin templates into a single-file per coin template to be used going forwards.  
+
+**Pre-reqs:**
+* manifest-latest.json
+* wallet-confs/*.conf
+* xbridge-confs/*.conf
+
+**Execution:**
+within your local repo/autobuild directory, for example:
+```
+mark@x230:~/src/blockdx-configs$ cd autobuild
+mark@x230:~/src/blockdx-configs/autobuild$ python3 create-j2-confs.py
+ABET,ABS,AEX,AGM,APR,ATB,AUS,BAD,BCD,BCH,BCZ,BIT,BITG,BLAST,BLOCK,BSD,BTC,BTDX,BTG,BTX,BZX,CARE,CDZC,CHC,CHI,CHN,CIV,CNMC,COLX,CRAVE,D,DASH,DGB,DIVI,DMD,DOGE,DOGEC,DSR,DVT,DYN,ECA,EMC,EMC2,ENT,FAIR,FGC,FJC,FLO,GALI,GBX,GEEK,GIN,GLC,GMCN,GXX,HASH,HATCH,HLM,HTML,INN,IOP,IXC,JEW,JIYOX,KLKS,KREDS,KYDC,KZC,LBC,LTC,LUX,LYNX,MAC,MLM,MNP,MONA,MRX,MUE,N8V,NIX,NMC,NOR,NORT,NYEX,NYX,ODIN,OHMC,OPCX,ORE,PAC,PART,PHL,PHR,PIVX,POLIS,PURA,QBIC,QTUM,RAP,REEX,RPD,RVN,SCC,SCN,SCRIBE,SEND,SEQ,SIB,SPK,STAK,SUB1X,SWIFT,SYS,TRB,TRC,UFO,UNO,VIA,VITAE,VIVO,VSX,VTC,WAGE,WGR,XC,XMCC,XMY,XN,XP,XVG,XZC,ZNZ
+```
+
+**Output:**
+* autobuild/configs/*.base.j2
+
+It addition to writing the j2 template files the script writes on the console the list of coins which were successfully processed.
+
+### utils/manifest-management.py
+**Purpose:**
+interactive manifest management
+
+**Pre-reqs:**
+* manifest-latest.json 
+
+**Execution:**
+within your local repo directory, for example:
+```
+mark@x230:~/src/blockdx-configs$ cd autobuild
+mark@x230:~/src/blockdx-configs/autobuild$ python3 manifest-management.py
+```
+
+**Output:**
+* manifest-latest.json
+
+**Warning:**
+this script is still WIP and not ready for use.
+
+
+
+# Workflow
+
+## Update coin configs
+**Audience:** everyone
+
+The new process to update a coin, eg: on a version upgrade is as follows:
+
+1. fork the https://github.com/blocknetdx/blockchain-configuration-files repo
+2. clone the repo to your local machine
+3. checkout a branch from master with a suitable name, eg: bump-thing-v4.2.2 
+4. update information in the manifest-latest.json
+5. update information in autobuild/configs/thing.base.j2
+6. run app.py as described above to update the {wallet,xbridge}-confs/thing.conf
+7. commit the changes to
+   * manifest-latest.json
+   * autobuild/configs/thing.base.j2
+   * wallet-confs/thing.conf
+   * xbridge-confs/thing.conf
+8. push the branch back to your repo
+
+Other parts of the workflow depend on who is performing them and take place in other repos but are documented here for simplicity in the order they are expected to happen.
+
+## Build a new Docker image
+**Audience:** 
+Not applicable to 3rd party developers who are simply looking to list a coin.
+
+* In the https://github.com/blocknetdx/dockerimages repo go to Actions and choose the BUILD IMAGE FROM TEMPLATE workflow. 
```

Comment:
Suggested change:
`In the https://github.com/blocknetdx/dockerimages repo go to Actions`
change to:
`Go to https://github.com/blocknetdx/dockerimages/actions`

Reply:
Done